pipelines/build/openjdk11_pipeline.groovy

```diff
@@ -166,6 +166,13 @@ def buildConfigurations = [
                 additionalFileNameTag: "linuxXL",
                 configureArgs        : '--with-noncompressedrefs --disable-ccache'
         ],
+        riscv64Linux      :  [
+                os                   : 'linux',
+                additionalNodeLabels : 'riscvcross'
+                arch                 : 'riscv'
+                test                 : ['sanity.openjdk', 'sanity.system', 'extended.system'],
```

Comment:
Pretty sure we didn't want testing so setting this to `test : []` will not run tests

Reply:
For now, I agree - We'll want to test it _eventually_ - but probably when the JDK itself is more stable :)